server/tools/sql_tables/Training.sql

```diff
@@ -0,0 +1,23 @@
+SET FOREIGN_KEY_CHECKS = 0;
+--
+-- Database: `cAuth`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `Training`
+--
+DROP TABLE IF EXISTS `Training`;
+CREATE TABLE `Training` (
+  `Location` varchar(255) NOT NULL,
+  `LocationDetail` varchar(255) NOT NULL,
+  `Day` varchar(255) NOT NULL,
+  `Time` time NOT NULL,
+  `UpdatedTime` timestamp NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+
+ALTER TABLE `Training`
+  ADD PRIMARY KEY (`UpdatedTime`);
```

Comment:
let's use the default ID (auto_increment) as the primary key. in the future we may want link this table to the other tables. for example, we can have an attendance table that links the training we the attendees. in that case, using id as primary key will be much easier.